package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "weather-today",
+  "version": "1.0.0",
+  "description": "Find out the weather information of your city using this simple weather app built with JavaScript.",
+  "private": true,
+  "scripts": {
+    "test": "jest",
+    "watch-jest": "jest --watchAll --passWithNoTests",
+    "dev": "webpack-dev-server --open",
+    "build": "webpack -p"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/johnsonsirv/weather-today.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/johnsonsirv/weather-today/issues"
+  },
+  "homepage": "https://github.com/johnsonsirv/weather-today#readme",
+  "devDependencies": {
+    "@babel/preset-env": "^7.7.6",
+    "clean-webpack-plugin": "^3.0.0",
+    "css-loader": "^3.3.2",
+    "eslint": "^6.7.2",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-import-resolver-webpack": "^0.12.0",
+    "eslint-plugin-import": "^2.19.1",
+    "file-loader": "^5.0.2",
+    "html-webpack-plugin": "^3.2.0",
+    "jest": "^24.9.0",
+    "style-loader": "^1.0.1",
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.3.10",
+    "webpack-dev-server": "^3.9.0"
+  },
+  "dependencies": {
+    "clean-webpack-plugin": "^3.0.0",
+    "css-loader": "^3.3.2",
+    "html-webpack-plugin": "^3.2.0",
```

Comment:
You have this and clean-webpack-plugin, as both a dependency and  a dev-dependency, please look into that

Reply:
> ### STATUS: CHANGES REQUIRED
> Hi @johnsonsirv you have done well working on this, please take note of the following changes required
> 
> * There should be a background change for every new search
>   ![screenshot1](https://camo.githubusercontent.com/6971d29699496a36de3c3872938304b5b2dafdb6/68747470733a2f2f6769746c61622e636f6d2f6d6963726f76657273652f6775696465732f70726f6a656374732f726571756972656d656e74735f73637265656e73686f74732f7261772f6d61737465722f696d616765732f6a6176617363726970742f776561746865725f6170702f617070656172616e63655f726571756972656d656e742e706e67)
> * Use either of promises or async/await
>   ![screenshot2](https://camo.githubusercontent.com/123ef19469efed5ff03bdd6cca8ff338b3600a7f/68747470733a2f2f6769746c61622e636f6d2f6d6963726f76657273652f6775696465732f70726f6a656374732f726571756972656d656e74735f73637265656e73686f74732f7261772f6d61737465722f696d616765732f6a6176617363726970742f776561746865725f6170702f70726f6d697365735f726571756972656d656e742e706e67)
> 
> And I think we can move forward for now, instead of being held back by the Eslint issue,  Also yes you are right the dist folder is being ignored by stickler so it cannot be the cause of the issue 
> 
> RAPHAEL NORIODE

Thank you Raphael. All these suggestions you mentioned are already in the app. I also pointed it out to @bellom who had earlier reviewed the code [Here](https://github.com/johnsonsirv/weather-today/pull/1).

Kindly use the [live link](https://js-weather-widget.netlify.com/) to see how it works.

**Also, my codebase uses a ``Promise`` to get data from the API.** 
![note-pullrequest](https://user-images.githubusercontent.com/25527349/71771955-5b578c80-2f43-11ea-86c2-d499c5a3bde5.PNG)